pom.xml

```diff
@@ -91,7 +91,7 @@
 		<dependency>
 			<groupId>commons-fileupload</groupId>
 			<artifactId>commons-fileupload</artifactId>
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `commons-fileupload` violates your company's policies.

Lifecycle recommends using version [1.4](https://search.maven.org/artifact/commons-fileupload/commons-fileupload/1.4/jar) instead as it does not violate any policies.